doc/releases/migration-guide-4.0.rst

```diff
@@ -41,6 +41,11 @@ Mbed TLS
 * The Kconfig options ``CONFIG_MBEDTLS_TLS_VERSION_1_0`` and ``CONFIG_MBEDTLS_TLS_VERSION_1_1``
   have been removed because Mbed TLS doesn't support TLS 1.0 and 1.1 anymore since v3.0. (:github:`76833`)
 
+* The Kconfig option ``CONFIG_MBEDTLS_SSL_EXPORT_KEYS`` was removed because the
+  corresponding build symbol was removed from Mbed TLS since revision 3.1.0.
```

Comment:
Rather either 
```suggestion
  corresponding build symbol was removed in Mbed TLS 3.1.0.
```
or
```suggestion
  corresponding build symbol doesn't exist anymore since Mbed TLS 3.1.0.
```
and up to you whether you want to add `version`.